IO.Eventuate.Tram.IntegrationTests/testsettings.json

```diff
@@ -1,6 +1,6 @@
 {
-  "KafkaBootstrapServers":  "kafka:9092",
+  "KafkaBootstrapServers":  "localhost:9092",
```

Comment:
Was the test transitioned from Docker to local? What about on the build machine?

Reply:
I think it was changed to allow the tests to be run on a dev's local machine from the IDE. When the tests are run within a docker container (as they are during the CI build), these settings are overridden via environment variables defined in the docker-compose.yml file.